content/docs/deployment.md

```diff
@@ -208,6 +239,37 @@ Shards and replicas are some configuration values to take special attention to,
 index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
 more information about choosing how many shards should be chosen for optimization.
 
+### Kafka
+Supported in Jaeger since 1.6.0
+Supported Kafka versions: 0.8+
```

Comment:
this looks like this:
![image](https://user-images.githubusercontent.com/3523016/44682065-5cec7880-aa10-11e8-915e-8f7a26aedbf2.png)

perhaps use bullets